objects/ArkhamSCE330-1092023-Page1.964222.json

```diff
@@ -0,0 +1,227 @@
+  "DragSelectable": true,
+  "GMNotes": "",
+  "GUID": "964222",
+  "Grid": true,
+  "GridProjection": false,
+  "Hands": false,
+  "HideWhenFaceDown": false,
+  "IgnoreFoW": false,
+  "LayoutGroupSortIndex": 0,
+  "Locked": false,
+  "LuaScript": "",
+  "LuaScriptState": "",
+  "MeasureMovement": false,
+  "Name": "Notecard",
+  "Nickname": "Arkham SCE 3.3.0 - 10/9/2023 - Page 1",
+  "Snap": true,
+  "States": {
+    "2": {
+      "AltLookAngle": {
+        "x": 0,
+        "y": 0,
+        "z": 0
+      },
+      "Autoraise": true,
+      "ColorDiffuse": {
+        "b": 1,
+        "g": 1,
+        "r": 1
+      },
+      "Description": "- Added the following community contributions:\n    Buffy the Vampire Slayer by AtomicZ!\n    City of Secrets by Exhaled Innards!\n    Circus Ex Mortis Investigator Expansion by The\n    Beard!\n    Heart of Darkness by Vinn Quest!\n    The Red Coterie Investigators by\n    Mattastrophic!\n    (continued)",
+      "DragSelectable": true,
+      "GMNotes": "",
+      "GUID": "d7faf7",
+      "Grid": true,
+      "GridProjection": false,
+      "Hands": false,
+      "HideWhenFaceDown": false,
+      "IgnoreFoW": false,
+      "LayoutGroupSortIndex": 0,
+      "Locked": false,
+      "LuaScript": "",
+      "LuaScriptState": "",
+      "MeasureMovement": false,
+      "Name": "Notecard",
+      "Nickname": "Arkham SCE 3.3.0 - 10/9/2023 - Page 2",
+      "Snap": true,
+      "Sticky": true,
+      "Tooltip": true,
+      "Transform": {
+        "posX": -26.88849,
+        "posY": 1.551499,
+        "posZ": -60.2882576,
+        "rotX": -1.53265116e-8,
+        "rotY": 90.0020447,
+        "rotZ": 3.44967188e-9,
+        "scaleX": 3,
+        "scaleY": 1,
+        "scaleZ": 3
+      },
+      "Value": 0,
+      "XmlUI": ""
+    },
+    "3": {
+      "AltLookAngle": {
+        "x": 0,
+        "y": 0,
+        "z": 0
+      },
+      "Autoraise": true,
+      "ColorDiffuse": {
+        "b": 1,
+        "g": 1,
+        "r": 1
+      },
+      "Description": "    The Sands of Memphis by Myriad!\n    Souls of Darkness by JackOfHearts!\n- The options panel has a new option to spawn clickable counters only on \"0 uses\" cards. Try the \"Chef's Selection\" option for the best of both worlds!\n- Increased readability of master Clue Counter's text.",
+      "DragSelectable": true,
+      "GMNotes": "",
+      "GUID": "50d85e",
+      "Grid": true,
+      "GridProjection": false,
+      "Hands": false,
+      "HideWhenFaceDown": false,
+      "IgnoreFoW": false,
+      "LayoutGroupSortIndex": 0,
+      "Locked": false,
+      "LuaScript": "",
+      "LuaScriptState": "",
+      "MeasureMovement": false,
+      "Name": "Notecard",
+      "Nickname": "Arkham SCE 3.3.0 - 10/9/2023 - Page 3",
+      "Snap": true,
+      "Sticky": true,
+      "Tooltip": true,
+      "Transform": {
+        "posX": -26.88849,
+        "posY": 1.551499,
+        "posZ": -60.2882576,
+        "rotX": -5.66725378e-9,
```

Comment:
Some of these seem to not have been correctly changed to 0

Reply:
Oh, good catch, let me try and fix that real quick.

Reply:
Okay, should be fixed, I think I got them all